tests/common/common.h

```diff
@@ -829,4 +829,29 @@ inline size_t linearize(sycl::range<3> range, sycl::id<3> id) {
   return id[2] + id[1] * range[2] + id[0] * range[1] * range[2];
 }
 
+/**
+Computes a multi-dimensional index such that id = unlinearize(range,
+linearize(range, id)) if id is a valid index in range. */
+template <unsigned int dimension>
+sycl::id<dimension> unlinearize(sycl::range<dimension> range, size_t id);
+
+inline sycl::id<1> unlinearize(sycl::range<1> range, size_t id) {
+  static_cast<void>(range);
```

Comment:
```suggestion
inline sycl::id<1> unlinearize(sycl::range<1>, size_t id) {
```

[`std::ignore`](https://en.cppreference.com/w/cpp/utility/tuple/ignore) is also an option.

My own suggestion is what I would choose myself, but I don't have strong preference, this is more of a FYI